members/B001267.yaml

```
@@ -50,7 +50,7 @@ contact_form:
         - name: Message
           selectors: [ "#field_0F40B55B-8631-4AA2-8223-08DE119CD370" ]
           values: [ "$MESSAGE" ]
-          commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t').replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC').replace(/[#]/g,'');"]
+          commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t').replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC').replace(/[#]/g,'').replace(/[(1)]/g, '');"]
```

```suggestion
          commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t').replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC').replace(/[#]/g,'').replace(/\\(1\\)/g, '');"]
```
The `[]` define character set, which you don't want, you want an exact pattern match. Since `( )` is used to define capture groups within regexes, you have to escape those characters. Need to double escape the `\` because yamls suck, if we weren't in yaml world the regex would be: `/\(1\)/g`

---

ty ty